enterprise/api/v1/views/enterprise_customer_catalog.py

```diff
@@ -76,6 +82,32 @@ def create(self, request, *args, **kwargs):
             enterprise_customer_uuid, enterprise_catalog_query_id)
         return super().create(request, *args, **kwargs)
 
+    def partial_update(self, request, *args, **kwargs):
+        """
+        Partially updates only the title in EnterpriseCustomerCatalog and returns the updated object.
+
+        URL: /enterprise/api/v1/enterprise-customer-catalog/
+
+        Method: PATCH
+
+        Payload::
+          {
+            "uuid": string - UUID of an existing enterprise customer catalog
+            "title":  string - Title of the catalog,
+          }
+
+        Returns 200 along with the updated object.
+        """
+        enterprise_customer_catalog_uuid = request.data.get('uuid')
+        found_catalog = self.has_enterprise_customer_catalog(enterprise_customer_catalog_uuid)
+        if not found_catalog:
+            return Response({'detail': 'Could not find catalog uuid'}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
I would put the uuid that wasn't found in the response

Reply:
good call!